russian/light-fire/brake-the-wall/i-am-the-bee.md

```diff
@@ -0,0 +1 @@
+Вы возвращаетесь обратно и осматриваете комнату, пытаясь найти хорошее укрытие.  Замечаете зеркало прислоненное к стене у окна. Непреодолимая сила тянет вас к нему. Вы подбегете к зеркалу, и ...О, нет! ...О, да! из зеркала на вас смотрит полосатая, пушистая пчела. Вы расправляете свои перламутровые крылышки и летите прочь от зефирного кошмара.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.